build-tools/packages/build-cli/src/commands/check/policy.ts

```diff
@@ -0,0 +1,262 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import * as fs from "fs";
+import { EOL as newline } from "os";
+import * as childProcess from "child_process";
+// eslint-disable-next-line unicorn/import-style
+import * as path from "path";
+import { Flags } from "@oclif/core";
+import { handlers } from "@fluidframework/build-tools";
+import { BaseCommand } from "../../base";
+
+const readStdin: () => Promise<string | undefined> = async () => {
+    return new Promise((resolve) => {
+        const stdin = process.openStdin();
+        stdin.setEncoding("utf-8");
+
+        let data = "";
+        stdin.on("data", (chunk) => {
+            data += chunk;
+        });
+
+        stdin.on("end", () => {
+            resolve(data);
+        });
+
+        if (stdin.isTTY) {
+            resolve("");
+        }
+    });
+};
+
+type policyAction = "handle" | "resolve" | "final";
+
+export class CheckPolicy extends BaseCommand<typeof CheckPolicy.flags> {
+    static description =
+        "Checks that the dependencies between Fluid Framework packages are following policies.";
+
+    static flags = {
+        fix: Flags.boolean({
+            description: `Fix errors if possible`,
+            required: false,
+            char: "f",
+        }),
+        handler: Flags.string({
+            description: `Filter handler names by <regex>`,
+            required: false,
+            char: "d",
+        }),
+        path: Flags.string({
+            description: `Filter file paths by <regex>`,
+            required: false,
+            char: "p",
+        }),
+        exclusions: Flags.string({
+            description: `Filter exclusions path`,
+            required: false,
+            char: "e",
+            default: "exclusions.json",
+        }),
+        stdin: Flags.boolean({
+            description: `Get file from stdin`,
+            required: false,
+        }),
+        ...BaseCommand.flags,
+    };
+
+    static handlerActionPerf = new Map<policyAction, Map<string, number>>();
+    static processed = 0;
+    static count = 0;
+    static pathToGitRoot = "";
+
+    async run() {
+        const handlerRegex: RegExp =
+            typeof this.processedFlags.handler === "string"
+                ? new RegExp(this.processedFlags.handler, "i")
+                : /.?/;
+        const pathRegex: RegExp =
+            typeof this.processedFlags.path === "string"
+                ? new RegExp(this.processedFlags.path, "i")
+                : /.?/;
+
+        // eslint-disable-next-line @typescript-eslint/no-require-imports, unicorn/prefer-module
+        const exclusions: RegExp[] = require(`../../data/${this.processedFlags.exclusions}`).map(
+            (e: string) => new RegExp(e, "i"),
+        );
```

Comment:
You need to use the exclusions flag as-is here, not append it to a path. The expectation is that the caller will provide a path to the exclusions file -- as we do when we call policy-check in our scripts -- and that if they do not, we will assume it's in the current working directory, if it's not there, the command will bail early. By changing the flag type to file above, oclif will handle all the checking for us.

Reply:
```suggestion
        const exclusions: RegExp[] = require(this.processedFlags.exclusions).map(
            (e: string) => new RegExp(e, "i"),
        );
```
Something like this